NEWS.md

```diff
@@ -2,6 +2,20 @@
 shiny 1.5.0.9000
```

Comment:
I think this should be changed to 1.6.0

Reply:
If this is prior to release and something downstream developers will see, we now write something like:

```
# shiny (development version)
(to be released as 1.6.0)
```

Otherwise we've found that people get confused about exactly which version they have vs what the news is talking about.

Reply:
Good to know, #3239 does the actual release bump, but I'll make a note to do that for > 1.6.0